core/domain/feedback_jobs_continuous.py

```diff
@@ -145,6 +145,38 @@ def get_thread_analytics(cls, exploration_id):
         return feedback_domain.FeedbackAnalytics(
             exploration_id, num_open_threads, num_total_threads)
 
+    @classmethod
+    def get_thread_analytics_multi(cls, exploration_ids):
+        realtime_model_ids = cls.get_multi_active_realtime_layer_ids(
+            exploration_ids)
+        realtime_models = cls._get_realtime_datastore_class().get_multi(
+            realtime_model_ids)
+        feedback_thread_analytics_models = (
+            feedback_models.FeedbackAnalyticsModel.get_multi(
+                exploration_ids))
+
+        num_open_threads_list = []
+        num_total_threads_list = []
+        for i in range(len(exploration_ids)):
+            num_open_threads = 0
+            num_total_threads = 0
+            if realtime_models[i] is not None:
```

Comment:
@seanlip - Sorry, but I fail to get the point of writing tests for this - since the current implementation clearly shows that the value will simply be taken as 0 in the case when either of the models is `None`. And when both are None - then it will yield `0 + 0 = 0` as the `num_open_threads` and `num_total_threads`. See return statement of get_therad_analytics_multi inside the feedback_jobs_continuous in [latest diff](https://github.com/oppia/oppia/pull/2096/files#diff-9d97b8f15dc4d5be390818933f4c5f49). Am I missing something 😄  ?